isolate_example/lib/page_two.dart

```diff
@@ -157,17 +164,27 @@ class RunningList extends StatelessWidget {
 
     return DecoratedBox(
       decoration: BoxDecoration(
-        color: (controller.running == true && controller.paused == false)
-            ? Colors.lightGreenAccent
-            : Colors.deepOrangeAccent,
+        color: Colors.grey[200],
       ),
       child: new ListView.builder(
         itemCount: sums.length,
         itemBuilder: (context, index) {
-          return ListTile(
-            title: Text((sums.length - index).toString() +
-                '. ' +
-                sums[index].toString()),
+          return Column(
+            children: [
+              Card(
+                child: ListTile(
+                  leading: Text('${sums.length - index}.'),
+                  title: Text(sums[index].toString()),
```

Comment:
This one can be interpolated as well:

`Text('${sums[index]}'),`

All it'd really be doing is calling `toString` for you, but it's generally considered more idiomatic to use interpolation and avoid using `toString`.